### PR TITLE
[11.0][MIG] Migrate hr_experience

### PR DESCRIPTION
**Pull request by ddufresne**

### CODE REVIEW

**Comment by ddufresne** [comment]
The travis issue is unrelated with the module.

I proposed a fix anyway in a seperate PR:
https://github.com/OCA/hr/pull/438

File: hr_experience/README.rst [hunk 1]
```diff
@@ -0,0 +1,112 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (feketemihai) [inline hr_experience/README.rst]: @ddufresne Thanks for the migration, can you please change to png files.

File: hr_experience/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Copyright 2013 Savoir-faire Linux (<http://www.savoirfairelinux.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Experience Management",
+    "version": "1.0.0",
```
Comment (feketemihai) [inline hr_experience/__manifest__.py]: Update version number

File: hr_experience/views/hr_certification_view.xml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        <field name="model">hr.certification</field>
+        <field name="type">tree</field>
+        <field name="arch" type="xml">
+            <tree string="certifications">
+                <field name="name" string="Certification"/>
+                <field name="employee_id"/>
+                <field name="certification" string="# Certificate"/>
+                <field name="partner_id" string="Issued By"/>
+                <field name="start_date"/>
+                <field name="end_date"/>
```
Comment (feketemihai) [inline hr_experience/views/hr_certification_view.xml]: Can you add expire field as well in tree view and adapt end_date domain.

File: hr_experience/views/hr_employee_view.xml [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+                        </field>
+                    </group>
+                    <group string="Certifications">
+                        <field name="certification_ids" context="{'default_employee_id': active_id}" nolabel="1">
+                            <tree string="Certifications">
+                                <field name="name" string="Certification"/>
+                                <field name="certification" string="# Certificate"/>
+                                <field name="partner_id" string="Issued By"/>
+                                <field name="start_date"/>
+                                <field name="end_date"/>
```
Comment (feketemihai) [inline hr_experience/views/hr_employee_view.xml]: The same, expire field and adapt end_date domain.

**Review by feketemihai — CHANGES_REQUESTED** [review]

**Comment by ddufresne** [comment]
@feketemihai all fixed!

**Comment by feketemihai** [comment]
@ddufresne Thanks, i will test on runbot when finished.

**Comment by ddufresne** [comment]
@feketemihai before you test, I am about to add some fixes on the xml views.

**Comment by ddufresne** [comment]
@feketemihai ok, you may check runbot. Thanks!

**Review by feketemihai — APPROVED** [review]

**Comment by foutoucour** [comment]
Hello here,
Would you have an ETA on the merge of this PR?

Best,
Jordi

**Comment by pedrobaeza** [comment]
@foutoucour each PR requires at least 2 approvals. Please make your review for getting this merged.

**Review by foutoucour — APPROVED** [review]

**Pull request merged**